java/src/Ice/src/main/java/com/zeroc/Ice/InputStream.java

```diff
@@ -1622,43 +1622,6 @@ public void readValue(java.util.function.Consumer<Value> cb) {
     readValue(cb, Value.class);
   }
```

Comment:
Please simplify the readValue above now that (presumably) the callback is never null.